services/snmptrap/service.go

```diff
@@ -138,8 +138,9 @@ func (s *Service) Trap(trapOid string, dataList []Data) error {
 		return errors.Wrapf(err, "invalid trap Oid %q", trapOid)
 	}
 	varBinds := snmpgo.VarBinds{
-		snmpgo.NewVarBind(snmpgo.OidSnmpTrap, oid),
+		snmpgo.NewVarBind(snmpgo.OidSysUpTime, snmpgo.NewTimeTicks(1000)),
 	}
+	varBinds = append(varBinds, snmpgo.NewVarBind(snmpgo.OidSnmpTrap, oid))
```

Comment:
No need to append here, you can simply declare them both in the literal array above.

```
varBinds := snmpgo.VarBinds{
		snmpgo.NewVarBind(snmpgo.OidSnmpTrap, oid),
		snmpgo.NewVarBind(snmpgo.OidSysUpTime, snmpgo.NewTimeTicks(1000)),
 }
```